compiler/src/iree/compiler/Utils/test/TileSizeSelectionTest.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "iree/compiler/Codegen/Common/TileSizeSelection.h"
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.h"
+
+using namespace mlir::iree_compiler;
+using namespace llvm;
+using namespace mlir;
```

Comment:
Instead of using-directives, I'd just wrap below into namespaces. Using-directives is dangerous to me, see https://abseil.io/tips/153 for more details.

> I view using-directives as time-bombs, both for the parties that deal in them and the language system. – Ashley Hedberg with apologies to Warren Buffett
>
> tl;dr
> Using-directives (using namespace foo) are dangerous enough to be banned by the [Google style guide](https://google.github.io/styleguide/cppguide.html#Namespaces). Don’t use them in code that will ever need to be upgraded.
>
> If you wish to shorten a name, you may instead use a namespace alias (namespace baz = ::foo::bar::baz;) or a using-declaration (using ::foo::SomeName), both of which are permitted by the style guide in certain contexts (e.g. in *.cc files).


```suggestion
namespace mlir::iree_compiler {
// ...
```